pennylane/ops/qubit/hamiltonian.py

```diff
@@ -209,6 +208,8 @@ def __init__(
         # this causes H.data to be a list of tensor scalars,
         # while H.coeffs is the original tensor
         super().__init__(*coeffs_flat, wires=self._wires, id=id, do_queue=do_queue)
+        # add to hyperparams so we can pass it to representations
+        self._hyperparameters["ops"] = self.ops
```

Comment:
I think this is something we'd want to do anyways in Josh's introduction of params and hyperparameters...pre-empted it here!